### PR TITLE
Add event to p_locale_xhtml()

### PR DESCRIPTION
**Pull request by annda**
Allows plugins to modify localized text

### CODE REVIEW

**Review by splitbrain — CHANGES_REQUESTED** [review]
Currently the event handler has no way of knowing which locale is being shown. The `$id` should be passed with the event data.

**Comment by annda** [comment]
The `$id` is now included.

File: inc/parserutils.php [hunk 1]
```diff
@@ -115,6 +115,8 @@ function p_wiki_xhtml($id, $rev='', $excuse=true,$date_at=''){
 function p_locale_xhtml($id){
     //fetch parsed locale
     $html = p_cached_output(localeFN($id));
+    $data = ['id' => $id, 'html' => $html];
+    Event::createAndTrigger('PARSER_LOCALE_XHTML', $data);
     return $html;
```
Comment (splitbrain) [inline inc/parserutils.php:120]: You return the original $html instead of the HTML modified by the plugin.

Comment (splitbrain) [reply inc/parserutils.php:120]: I also wonder if the event shouldn't wrap around `p_cached_output(localeFN())` so plugins can also change the `$id` in the `BEFORE` event?

**Comment by splitbrain** [comment]
@annda Can you update the docs please?

**Pull request merged**